package.json

```diff
@@ -20,6 +20,7 @@
     "pluralize": "^8.0.0",
     "popper.js": "^1.16.1",
     "prop-types": "^15.7.2",
+    "qs": "^6.9.4",
```

Comment:
We already have `query-string` as a dependency, is `qs` really needed?

Reply:
I'll try to change it to use `query-string` then. :wink: